llvm/lib/Target/SPIRV/SPIRVModuleAnalysis.cpp

```diff
@@ -1714,6 +1724,19 @@ void addInstrRequirements(const MachineInstr &MI,
       Reqs.addCapability(SPIRV::Capability::StorageImageWriteWithoutFormat);
     break;
   }
+  case SPIRV::OpTypeStructContinuedINTEL:
+  case SPIRV::OpConstantCompositeContinuedINTEL:
+  case SPIRV::OpSpecConstantCompositeContinuedINTEL:
+  case SPIRV::OpCompositeConstructContinuedINTEL: {
```

Comment:
Are those instructions tested elsewhere?

Reply:
I'll add them in follow-up PRs - for now I just added the common infrastructure for this extension